xml/security_ldap.xml

```diff
@@ -41,15 +36,11 @@
  </para>
  <itemizedlist mark="bullet" spacing="normal">
   <listitem>
-   <remark condition="clarity">
-    2011-02-18 - fs: the number is limited? and why limit reads?
-   </remark>
    <para>
-    To make multiple concurrent reading accesses possible, the number of
-    updates is usually very low. The number of read and write accesses is
-    often limited to a few users with administrative privileges. In
-    contrast, conventional databases are optimized for accepting the largest
-    possible data volume in a short time.
+    The type of data stored in a directory tends to be long lived and changes
+    infrequently. This allows the service to be optimized for high performance
+    concurrent reads. In contrast, conventional databases are optimized for
+    accepting the largest possible data volume in a short time.
```

Comment:
accepting a large number of writes to data in a short time?

Reply:
Argh, overlooked that at first. Now in.